apb_overview.md

```diff
@@ -8,39 +8,38 @@ This document outlines the steps for each APB task to help people to understand
     * Secrets for database admin users
     * Service for the Postgres DB
     * PersistVolume Claim to for the Postgres DB
-    * DeploymentConfig for the Postgres DB
+    * Deployment for the Postgres DB
 2. Create a new Keycloak service, including:
-    * DeploymentConfig for Keycloak
+    * Deployment for Keycloak
         * Link Keycloak and Postgres DB
         * Add default Keycloak admin user credential
     * Service for Keycloak
     * Routes (either HTTP or HTTPS depends on the option) for Keycloak
     * Create a new realm in Keycloak using the current OpenShift project name
-    * Create a new public Keycloak client (with generated name and password) in the newly created realm
-    * Create a new bearer-only Keycloak client with the same generated name and password in the newly created realm
-    * Create a new secret in the current OpenShift namespace with the following info:
-        * URL of the Keycloak service
-        * Admin user credentials
+    * Create a new secret in the current OpenShift namespace with admin user credentials
+    * Create a new configmap (labelled with the Keycloak service instance ID) in the current namespace with the following info:
+        * Name and type of the service
         * Realm name
-        * Details about the newly created public Keycloak client
-        * Details about the newly created bearer-only Keycloak client
+        * URL of the Keycloak service
     * Persist the admin user credential as `_apb_provision_creds`
 3. Create a new Keycloak-metrics service, including:
-    * DeploymentConfig for Keycloak-metrics
+    * Deployment for Keycloak-metrics
     * Persistent volume claim for Keycloak-metrics
     * Service for Keycloak-metrics
     * Copy `keycloak-monitoring-prometheus.jar` to Keycloak to collect metrics data about Keycloak
 
 # Bind
 
 1. Get the Keycloak admin user token using the Keycloak admin user credential
-2. Create a new bearer-only client with a newly generated name and password in Keycloak. It is created in the realm that matches the current namespace.
-3. Persist the newly created client name and password as `_apb_bind_creds`
+2. Create a new bearer-only client with a generated ID and password in Keycloak. It is created in the realm that matches the current namespace
```

Comment:
So this will need to change to say if the type is bearer then it creates a bearer client if the type is public it creates a new public client